content/en/monitors/monitor_types/outlier.md

```diff
@@ -16,133 +16,89 @@ further_reading:
-
-This implementation of DBSCAN takes one parameter, `tolerance`, the constant by which the initial threshold is multiplied to yield DBSCAN's distance parameter 𝜀.
+#### Algorithms
 
-{{< img src="monitors/monitor_types/outliers/outliers-dbscan-config.png" alt="outliers dbscan configuration" responsive="true" style="width:80%;">}}
+Datadog offers two types of outlier detection algorithms: `DBSCAN`/`scaledDBSCAN` and `MAD`/`scaledMAD`. It is recommended to use the default algorithm, DBSCAN. If you have trouble detecting the correct outliers, adjust the parameters of DBSCAN or try the MAD algorithm. The scaled algorithms may be useful if your metrics are large scale and closely clustered.
 
-Here is DBSCAN with a tolerance of 3.0 in action on a pool of Cassandra workers:
+##### DBSCAN
 
-{{< img src="monitors/monitor_types/outliers/outliers-dbscan-cassandra.png" alt="outliers dbscan cassandra" responsive="true" style="width:80%;">}}
+[DBSCAN][3] (density-based spatial clustering of applications with noise) is a popular clustering algorithm. Traditionally, DBSCAN takes: 1) a parameter 𝜀 that specifies a distance threshold under which two points are considered to be close; and 2) the minimum number of points that have to be within a point's 𝜀-radius before that point can start agglomerating. 
 
-Set the tolerance parameter according to how similarly you expect your hosts to behave—larger values allow for more tolerance in how much a host can deviate from its peers.
+Datadog uses a simplified form of DBSCAN to detect outliers on timeseries. Each host is considered to be a point in *d*-dimensions, where *d* is the number of elements in the timeseries. Any point can agglomerate, and any point not in the largest cluster is considered an outlier. The initial distance threshold is set by creating a new median timeseries by taking the median of the values from the existing timeseries at every time point. The Euclidean distance between each host and the median series is calculated. The threshold is set as the median of these distances, multiplied by a normalizing constant.
 
-### MAD
+**Parameters**<br>
+This implementation of DBSCAN takes one parameter, `tolerance`, the constant by which the initial threshold is multiplied to yield DBSCAN's distance parameter 𝜀. Set the tolerance parameter according to how similarly you expect your hosts to behave—larger values allow for more tolerance in how much a host can deviate from its peers.
```

Comment:
```suggestion
This implementation of DBSCAN takes one parameter, `tolerance`, the constant by which the initial threshold is multiplied to yield DBSCAN's distance parameter `𝜀`. Set the tolerance parameter according to how similarly you expect your tag values to behave—larger values of `𝜀` allow for more tolerance in how much a tag value can deviate from its peers.
```

Reply:
Would need to be verified by monitors

Reply:
Same here it would need to be `tag`.

Reply:
@unclebconnor - Do you have any feedback here or should we ask data science?

Reply:
DS might have something to add but this seems very clear to me.

Reply:
Same comment as above...I think `groups` or `tag groups` would be the way to go based on the language we use internally, assuming that's consistent with our external doc practices